modules/ocf_kubernetes/manifests/master/ingress/nginx.pp

```diff
@@ -16,11 +14,15 @@
     '/etc/ocf-kubernetes/manifests/ingress/ingress-expose.yaml':
       content => template('ocf_kubernetes/ingress/ingress_expose.yaml.erb'),
       mode    => '0644';
+
+    '/etc/ocf-kubernetes/manifests/ingress/ingress-deploy.yaml':
+      content => 'puppet:///modules/ocf_kubernetes/ingress_deploy.yaml',
```

Comment:
This sets the content of the file to literally `puppet:///modules/ocf_kubernetes/ingress_deploy.yaml`; correct attribute is `source`